x-pack/legacy/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/policy_table.js

```diff
@@ -299,6 +299,22 @@ export class PolicyTable extends Component {
     const { name } = policy;
     const cells = Object.entries(COLUMNS).map(([fieldName, { width }]) => {
       const value = policy[fieldName];
+
+      if (fieldName === 'name') {
+        return (
+          <th
```

Comment:
Ideally, EUI would publish a component we can use here, or provide props on `EuiTableRowCell` that allow it to provide the accessibility affordances you're adding here. Would you mind creating an issue and cross-linking to this code as an example?

Reply:
Yeah, agreed. I was looking through the existing EUI issues before I went ahead and opened one and it looks like you created a similar/related issue in the past 😀

See: https://github.com/elastic/eui/issues/1030

Reply:
Hmm, I'm not sure if that issue covers the requirements you're meeting with this particular change. If I understand correctly, that issue is about announcing each column header before reading the content of a particular cell. This helps users understand the meaning of the cell's content. For example, if a cell contained an address "123 ABC Street", and that column header was "Address", the screen-reader would announce "Address: 123 ABC Street".

But I believe the problem raised in https://github.com/elastic/kibana/issues/43842 was that the "Name" column (or some other identifying column) is also used to identify each row. This provides additional context so that you know which entity each cell is referring to. In my example above, if there was also a "Name" column and the content for that row was "Anastasia Romanov" then it sounds like somehow the screen-reader would connect "Anastasia Romanov" and "123 ABC Street". But to be honest, I'm not sure I understand how that works.

@barlowm Could you please clarify this for us?